.github/auto-comment.yml

```diff
@@ -0,0 +1,12 @@
+# Add comment related to interrupts action on new issues.
+issueOpened: >
+  Thank your for opening an issue. Our team's interrupts engineer will reivew your issue shortly.
+  
+   **Issue Resolution:**
+   - [ ] _[Interrupts Engineer]_  Triage / apply categorization labels
+   - [ ] _[Interrupts Engineer]_ Verify / Reproduce the reported issue
+   - [ ] _[Forseti Engineer]_ Perform root cause analysis
+   - [ ] _Forseti Engineer]_ Add tasks and next steps to resolve this issue.
+
+#pullRequestOpened: >
```

Comment:
Commented lines are not usually checked out.  So, should these 2 lines be here?

Reply:
I left this here if we wanted to later enable comments on PRs. Happy to remove them if that is a concern.